added default radius as inf

PetervDooren opened the pull request:
merge after https://github.com/tue-robotics/ed/pull/94 and https://github.com/tue-robotics/geolib2/pull/34

LarsJanssenTUe commented on robot_skills/src/robot_skills/world_model_ed.py:
```diff
@@ -26,6 +26,7 @@
 from robot_skills.util.entity import from_entity_info
 from robot_skills.robot_part import RobotPart
 
+infradius = 200000
```
Why this magic number?

PetervDooren replied:
Because while there are symbols for infinity in both python and c++. I don't think it is possible to send it over a rosmessage

MatthijsBurgh replied:
Infiniti can be send in a ros msg.

PetervDooren replied:
Fixed the magic number both here and in https://github.com/tue-robotics/ed/pull/94

PetervDooren commented:
Added a few more changes to this PR. Changed the emptyspotdesignator to work with our new query strategy. And noticed the example for the place designator did not work so I fixed that

LarsJanssenTUe commented on robot_smach_states/src/robot_smach_states/manipulation/place_designator.py:
```diff
@@ -89,9 +89,10 @@ def _resolve(self):
         return None
 
-    def is_poi_occupied(self, frame_stamped):
+    def is_poi_occupied(self, frame_stamped, surface_entity):
```
is_poi_unoccupied
